.github/workflows/ci_release.yml

```diff
@@ -36,6 +36,9 @@ jobs:
         uses: SkynetLabs/.github/.github/actions/yarn-basic-checks@master
         with:
           node-version: ${{ matrix.node-version }}
+      
+      - name: "Report test coverage impact"
```

Comment:
nit: this could probably be a standalone job since it doesn't need the tests to run first right? or does it?

Reply:
It does, tests generate a report and place it in `coverage` directory.